docs/migrations/rest.md

```diff
@@ -0,0 +1,87 @@
+# REST Endpoints Migration
+
+Migrate your REST endpoints to the Stargate ones. {synopsis}
+
+## Deprecation of Legacy REST Endpoints
+
+The Cosmos SDK versions v0.39 and earlier provided REST endpoints to query the state and broadcast transactions. These endpoints are kept in Cosmos SDK v0.40 (Stargate), but they are marked as deprecated, and will be removed in v0.41. We therefore call these endpoints legacy REST endpoints.
+
+Some important information concerning all legacy REST endpoints:
+
+- Most of these endpoints are backwards-comptatible. All breaking changes are described in the next section.
+- In particular, these endpoints still output Amino JSON. Cosmos v0.40 introduced Protobuf as the default encoding library throughout the codebase, but legacy REST endpoints are one of the few places where the encoding is hardcoded to Amino. For more information about Protobuf and Amino, please read our [encoding guide](../core/encoding.md).
+- All legacy REST endpoints include a [HTTP deprecation header](https://tools.ietf.org/id/draft-dalal-deprecation-header-01.html) which links to this document.
+
+## Breaking Changes in Legacy REST Endpoints
+
+| Legacy REST Endpoint      | Description        | Breaking Change                                                                                                                                                                                                                            |
+| ------------------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `GET /txs/{hash}`         | Query tx by hash   | Endpoint will error when trying to output non-Amino txs (e.g. IBC txs).                                                                                                                                                                    |
+| `GET /txs`                | Query tx by events | Endpoint will error when trying to output non-Amino txs (e.g. IBC txs).                                                                                                                                                                    |
```

Comment:
> Endpoint will error when trying to output non-Amino txs (e.g. IBC txs).                       

This doesn't sound clear to me. Can we reword it and describe what are the _non amino txs_?

Reply:
makes sense 👍